### PR TITLE
Support toggles of collectinput

### PR DESCRIPTION
**Pull request by ianlin-bbpos**
## Summary

Support collect inputs with toggles in the RN SDK.

## Testing

- [x] I tested this manually
- [ ] I added automated tests

## Documentation

Select one:

- [ ] I have added relevant documentation for my changes.
- [x] This PR does not result in any developer-facing changes.

### CODE REVIEW

File: dev-app/src/screens/CollectInputsScreen.tsx [hunk 1]
```diff
@@ -88,6 +89,13 @@ export default function CollectInputsScreen() {
                   description:
                     'Please sign if you agree to the terms and conditions',
                   submitButtonText: 'submit signature',
+                  toggles: [
+                    {
+                      title: 'fefef',
+                      description: 'essdsde',
```
Comment (nazli-stripe) [inline dev-app/src/screens/CollectInputsScreen.tsx:95]: is it okay to have placeholders here or should we switch it to real text?

File: android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt [hunk 2]
```diff
@@ -819,66 +821,179 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
...
+                                        toggle.getString("description"),
+                                        if (toggle.getString("defaultValue") == "ENABLED") {
+                                            ToggleValue.ENABLED
+                                        } else {
+                                            ToggleValue.DISABLED
+                                        }
+                                    )
+                                )
+                            }
+                        }
```
Comment (sjl-stripe) [inline android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt:841]: If feasible, refactor this into a method to reduce code duplication

File: dev-app/src/screens/CollectInputsScreen.tsx (hunk 1 above)
Comment (sjl-stripe) [reply dev-app/src/screens/CollectInputsScreen.tsx:95]: +1, we should switch to real text

File: ios/StripeTerminalReactNative.swift [hunk 3]
```diff
@@ -968,12 +968,29 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
...
+                                let description = it["description"] as! String
+                                let defaultValue = it["defaultValue"] as! String
+                                let toggle = try ToggleBuilder(defaultValue: (defaultValue == "ENABLED") ? ToggleValue.enabled : ToggleValue.disabled).setTitle(title).setStripeDescription(description).build()
+                                toggles.append(toggle)
+                            } catch {
+                                resolve(Errors.createError(nsError: error as NSError))
+                                return
+                            }
+                        }
+                    }
```
Comment (sjl-stripe) [inline ios/StripeTerminalReactNative.swift:986]: If feasible, refactor this into a method to reduce code duplication

File: dev-app/src/screens/CollectInputsScreen.tsx (hunk 1 above)
Comment (ianlin-bbpos) [reply dev-app/src/screens/CollectInputsScreen.tsx:95]: I referred to the examples in the document [https://docs.stripe.com/terminal/features/collect-inputs], no title and description. These two above were for debugging and not removed yet..

**Review by nazli-stripe — APPROVED** [review]

**Review by sjl-stripe — APPROVED** [review]

**Review by nazli-stripe — APPROVED** [review]

**Pull request merged**